### PR TITLE
Define AOM profile

### PR DESCRIPTION
**Pull request by cconcolato**
This PR proposes a profile for documents written by the [Alliance for Open Media](http://aomedia.org/). It is based on the existing profiles (e.g. DINI) and some cleaning. This is an initial version, likely to be updated in the future, but already good to use I think.

### CODE REVIEW

File: examples/example_aom.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        }],
+        formerEditors: [{
+          name: "Your Name",
+          url: "https://your-site.com",
+        }, {
+          name: "Your Name 2",
+          url: "https://your-site.com",
+        }],
+        publishDate: "2021-04-23",
+        github: "https://github.com/AOMedia/some-spec/"
```
Comment (marcoscaceres) [inline examples/example_aom.html:33]:
```suggestion
        // Defaults to 'gh-pages' branch.  
        // See https://respec.org/docs/#github if you need a different branch (e.g., "main"). 
        github: "AOMedia/some-spec"
```

File: examples/example_aom.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      </p>
+      <p>
+        ReSpec will list this reference as normative, like [[AV1]]. You SHOULD use references.
+      </p>
+      <p>
+        This is an example of a non-normative reference in a normative section
+        [[?JPEG]]
+      </p>
+    </section>
+    <section id="logo">
```
Comment (marcoscaceres) [inline examples/example_aom.html:70]: Maybe go with:  
```suggestion
    <section id="example-figure">
```

File: examples/example_aom.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        Figure
+      </h2>
+      <figure id="figure">
+        <img src="https://aomedia.org/assets/images/aomedia-icon-only.png" alt="AOM Logo">
+        <figcaption>
+          The AOM logo
+        </figcaption>
+      </figure>
+      <p>
+        Link to <a href="#figure"></a>
```
Comment (marcoscaceres) [inline examples/example_aom.html:81]:
```suggestion
        Link to [[[#figure]]] ... the three square-brackets "expand" a local reference.   
```

File: examples/example_aom.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      </p>
+    </section>
+    <section id="tof" class="informative appendix"></section>
+    <section id="index" class="informative appendix"></section>
```
Comment (marcoscaceres) [inline examples/example_aom.html:85]: I think 
```suggestion
    <section id="tof" class="appendix">
      <!-- a table of figures will be added here -->
    </section>
    <section id="index" class="appendix">
      <!-- Index of terms defined/used by this specification will be added here  -->
    </section>
```

File: src/aom/defaults.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      short: "AOM",
+      url: "http://aomedia.org/license/",
+    },
+  ],
+]);
+
+const aomDefaults = {
+  // treat document as "Common Markdown" (with a little bit of HTML).
+  // choice between Markdown and HTML depends on the complexity of the spec
+  // example of Markdown spec: https://github.com/WICG/netinfo/blob/gh-pages/index.html
```
Comment (marcoscaceres) [inline src/aom/defaults.js:22]:
```suggestion
  // example of Markdown spec: https://github.com/WICG/netinfo/blob/gh-pages/index.html
  // Helpful guide: https://respec.org/docs/#markdown
```

File: src/aom/headers.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+//         Example of usage: [{key: "foo", href:"https://b"}, {key: "bar", href:"https://"}].
+//         Allowed values are:
+//          - key: the key for the <dt> (e.g., "Bug Tracker"). Required.
+//          - value: The value that will appear in the <dd> (e.g., "GitHub"). Optional.
+//          - href: a URL for the value (e.g., "https://foo.com/issues"). Optional.
+//          - class: a string representing CSS classes. Optional.
+//  - license: can be one of the following
+//      - "cc-by-sa"
+//      - "cc-by"
+//      - "cc0"
```
Comment (marcoscaceres) [inline src/aom/headers.js:44]: You only allow "aom", right? 
```suggestion
//      - "aom"
```

File: src/aom/templates/headers.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+  if (existingCopyright) {
+    existingCopyright.remove();
+    return existingCopyright;
+  }
+  if (conf.hasOwnProperty("overrideCopyright")) {
+    const msg = "The `overrideCopyright` configuration option is deprecated.";
+    const hint = 'Please use `<p class="copyright">` instead.';
+    showWarning(msg, name, { hint });
+  }
+  return conf.overrideCopyright
```
Comment (marcoscaceres) [inline src/aom/templates/headers.js:100]: I would recommend not supporting `.overrideCopyright` In the `conf` object. 

To override the copyright, one can do:
```
<p id="copyright">
  No copyleft intended  2020!
</p>
```

https://respec.org/docs/#copyrights-patents

File: src/aom/headers.js [hunk 8]
```diff
@@ -0,0 +1,155 @@
+// @ts-check
+// Module aom/headers
+// Generate the headers material based on the provided configuration.
+// CONFIGURATION
+//  - specStatus: the short code for the specification's maturity level or type (required)
```
Comment (marcoscaceres) [inline src/aom/headers.js:5]: If you would like, we can add AOM specific section to the documentation. This is what we do for w3c:

https://respec.org/docs/#w3c-specific-configuration-options

**Review by marcoscaceres — COMMENTED** [review]
Looking pretty good @cconcolato! Couple of suggestions.

File: src/aom/templates/headers.js (hunk 7 above)
Comment (cconcolato) [reply src/aom/templates/headers.js]: fixed.

**Comment by cconcolato** [comment]
@marcoscaceres @sidvishnoi Node CI seems to be stuck with `GitHub Actions has encountered an internal error when running your job.` Any idea what's wrong?

**Comment by sidvishnoi** [comment]
@cconcolato GitHub actions are having a bad time today: https://githubstatus.com
We can try tomorrow.

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Adding some tests...

**Comment by marcoscaceres** [comment]
Moving everything to https://github.com/w3c/respec/pull/3543, as I added some tests.

**Pull request closed without merging**